.github/pull_request_template.md

```diff
@@ -1,24 +1,31 @@
 <!--- Provide a summary of your changes in the title field above. -->
 
+# Pull request
+
 ## Description
+
 <!--- Describe your changes. -->
 
 ## Motivation & context
+
 <!--- What problem does this change solve? -->
 <!--- Provide a link if you are addressing an open issue. -->
 
 ## Issue type checklist
+
 <!--- What type of change are you submitting? Check the boxes that apply: -->
+
 - [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
 - [ ] **New feature**: non-breaking change that adds functionality.
 - [ ] **Breaking change**: fix or feature that will cause current functionality to change.
```

Comment:
What about **Chores**?  There are significant improvements made that aren't really a bug fix, feature, or breaking change.  They are things user's probably wouldn't want to know about and so updates to these do not get documented in change logs.  Previously, we had types that are required to be used in titles. fix:, feat:, chore:.  Breaking was called out separately because it could be a fix or feature and breaking change.  Also, in the <footer> was where we would denote this PR fixes #1238  for example.  Where is this documented now?  Are we ok, that GitHub users should know this on their own?

Reply:
Changes made.